docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2915 @@
+
+
+Monitoring::
+Collect metrics about the active/idle connections to ES nodes {kibana-pull}141434[#141434]
+
+Observability::
+* Integrate alert search bar on rule details page {kibana-pull}144718[#144718]
+* Adds additional context to recovered alerts of Infrastructure rules {kibana-pull}144683[#144683]
+* Adds list of containers in context variable of Inventory rule {kibana-pull}144526[#144526]
+* Adds new contextual attributes to Infrastructure - Metric threshold rule {kibana-pull}143001[#143001]
+* Adds alert details page feature flag by App {kibana-pull}142839[#142839]
+* Adds new contextual attributes to Infrastructure - Inventory Rule {kibana-pull}140598[#140598]
+
+Platform::
+Adds notifications plugin, offering basic email service {kibana-pull}143303[#143303]
+
+Security::
+Adds the ability to show sub-feature privileges when using the Basic license {kibana-pull}142020[#142020]
+
+Uptime::
+Adds `created_at` field in saved objects {kibana-pull}143507[#143507]
+
+For more information about the features introduced in 8.6.0, refer to <<whats-new,What's new in 8.6>>.
+
+[[enhancements-and-bug-fixes-v8.6.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.6.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.6.0]]
+=== Enhancements
+Alerting::
+* Clone rule {kibana-pull}144741[#144741]
+* Remove errors and warning in triggers_actions_ui jest test {kibana-pull}144443[#144443]
+
+* Increase the default table size of the cases table to 10 {kibana-pull}144228[#144228]
+* Bulk enable rules api {kibana-pull}144216[#144216]
+* Create bulk delete on rules front {kibana-pull}144101[#144101]
+* Improve Task Manager’s retry logic for ad-hoc tasks {kibana-pull}143860[#143860]
+* Increases the max length limit of the case title to 160 characters {kibana-pull}143664[#143664]
+* Adds the ability to bulk edit tags in the cases table {kibana-pull}143450[#143450]
+* Filter cases without assignees {kibana-pull}143390[#143390]
+* Make actions retry when encountering failures {kibana-pull}143224[#143224]
+* Adds a backlink to cases when pushing in external services {kibana-pull}143174[#143174]
+* Move Connectors to own page {kibana-pull}142485[#142485]
+* 142183 create bulk delete on rules {kibana-pull}142466[#142466]
+* Allow `_source` field for ES DSL query rules {kibana-pull}142223[#142223]
+* Update rule status {kibana-pull}140882[#140882]
+
+APM::
+* Mobile most used pie charts with Lens {kibana-pull}144232[#144232]
+* Multi metric partition charts {kibana-pull}143966[#143966]
+* Adds missing options to APM central config {kibana-pull}143668[#143668]
+* Adds waterfall to dependency operations {kibana-pull}143257[#143257]
+* Display configuration table above code sample {kibana-pull}143178[#143178]
+* Implement improvements to AWS Lambda metrics view {kibana-pull}143113[#143113]
+* Display featured integrations at top of integrations list per category {kibana-pull}142771[#142771]
```

Comment:
```suggestion
APM::
* Adds pie charts displaying the most used mobile devices, operating systems, etc. {kibana-pull}144232[#144232]
* Adds new options to APM central configuration {kibana-pull}143668[#143668]
* Adds a trace waterfall to the dependency operation detail view {kibana-pull}143257[#143257]
* Adds a configuration table above code sample in getting started guide {kibana-pull}143178[#143178]
* Adds improvements to the AWS Lambda metrics view {kibana-pull}143113[#143113]
```

Reply:
Can you add these:

- https://github.com/elastic/kibana/pull/144172
- https://github.com/elastic/kibana/pull/144127
- https://github.com/elastic/kibana/pull/143735 [This is a tech preview item, but I think we should add it to the release notes anyway]
- https://github.com/elastic/kibana/pull/143179
- https://github.com/elastic/kibana/pull/143844 [This is also a tech preview]

Reply:
The mobile issues could also be combined -  we have now introduces a service overview page for mobile apps - https://github.com/elastic/apm-dev/issues/823